ivadomed/training.py

```diff
@@ -54,8 +55,8 @@ def train(model_params, dataset_train, dataset_val, training_params, path_output
     writer = SummaryWriter(log_dir=path_output)
 
     # BALANCE SAMPLES AND PYTORCH LOADER
-    conditions = all([training_params["balance_samples"]["applied"], model_params["name"] != "HeMIS"])
-    sampler_train, shuffle_train = get_sampler(dataset_train, conditions, training_params['balance_samples']['type'])
+    conditions = all([training_params["balance_samples"][BalanceSamplesKW.APPLIED], model_params[ModelParamsKW.NAME] != "HeMIS"])
+    sampler_train, shuffle_train = get_sampler(dataset_train, conditions, training_params['balance_samples'][BalanceSamplesKW.TYPE])
```

Comment:
You missed a few keywords here. "balanced_samples" is just from the context JSON. 

Also, I this training_params is just from the context dict, as this is the only place it was called: 
![image](https://user-images.githubusercontent.com/6720068/138386867-0609fe8b-26a0-405a-a1af-42f080ff17b9.png)

Reply:
Done